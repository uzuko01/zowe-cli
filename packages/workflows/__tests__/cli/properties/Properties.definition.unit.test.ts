import { ICommandDefinition } from "@brightside/imperative";

describe("zos-workflows list-properties definition", () => {
    it ("should not have changed", () => {
        const definition: ICommandDefinition = require("../../../src/cli/properties/Properties.definition").PropertiesDefinition;
        expect(definition).toBeDefined();
        expect(definition.children.length).toBe(1);
        delete definition.children;
        expect(definition).toMatchSnapshot();
    });
});
