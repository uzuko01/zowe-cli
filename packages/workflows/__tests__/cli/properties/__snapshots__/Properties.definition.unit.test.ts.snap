// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`zos-workflows list-properties definition should not have changed 1`] = `
Object {
  "aliases": Array [
    "lp",
  ],
  "description": "Get the properties of a workflow instance in z/OSMF",
  "name": "list-properties",
  "type": "group",
}
`;
